pkg/util/csi_proxy_v1beta_windows.go

```diff
@@ -0,0 +1,96 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"fmt"
+
+	volumeapi "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1beta2"
+	volumeclient "github.com/kubernetes-csi/csi-proxy/client/groups/volume/v1beta2"
+)
+
+// CSIProxyV1Beta is the CSI Proxy implementation that uses the v1 API
```

Comment:
You mean use v1beta2 API?
Also do you still need beta API support?

Reply:
we don't need it however it was useful to test against a GKE cluster that had v1beta still, I can remove it afterwards if it's not needed